access/common/auth/token_provider_test.go

```diff
@@ -0,0 +1,169 @@
+package auth
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/gravitational/teleport-plugins/access/common/auth/state"
+	"github.com/gravitational/trace"
+	"github.com/jonboulle/clockwork"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/suite"
```

Comment:
Is there a reason why you chose to use suite style tests here? I believe we are still trying to migrate our tests to simpler `testify/require` assertions as opposed to suites, using built in go test buildup/breakdown mechanisms instead - https://github.com/gravitational/teleport/blob/master/rfd/0001-testing-guidelines.md#details

Let me know if this RFD has fallen out of date and we have started using suite style tests again.

Reply:
Ah, I mostly structured this after existing tests. Will move to vanilla `testing.T` :+1: